source/extensions/filters/udp/udp_proxy/udp_proxy_filter.h

```diff
@@ -84,7 +91,10 @@ class UdpProxyFilterConfig {
     }
   }
 
-  const std::string& cluster() const { return cluster_; }
+  const std::string cluster(Network::Address::InstanceConstSharedPtr address) const {
```

Comment:
Should this perhaps be named "clusterForAddress()" or maybe "route()"?

Reply:
Ok, renamed to `route`.